experimental/traffic-portal/src/app/shared/tp-header/tp-header.service.spec.ts

```diff
@@ -15,7 +15,7 @@ import { TestBed } from "@angular/core/testing";
 
 import {TpHeaderComponent} from "src/app/shared/tp-header/tp-header.component";
 
-import { TpHeaderService } from "./tp-header.service";
+import {HeaderNavigation, TpHeaderService} from "./tp-header.service";
```

Comment:
nit: `HeaderNavigation` is only used as a type, could be a `type` import

although if you take my earlier advice and have the service own its default links this import will be unnecessary anyway.